docs/endpoints/openrtb2/amp.md

```diff
@@ -29,5 +29,22 @@ A sample response payload looks like this:
 }
 ```
 
+### Query Parameters
+
+This endpoint supports the following query parameters:
+
+1. `h` - `amp-ad` `height`
+2. `w` - `amp-ad` `width`
+3. `oh` - `amp-ad` `data-override-height`
+4. `ow` - `amp-ad` `data-override-width`
+5. `ms` - `amp-ad` `data-multi-size`
+6. `curl` - the canonical URL of the page
+7. `purl` - the page URL
+8. `timeout` - the publisher-specified timeout for the RTC callout
+   - A configuration option `amp.default-timeout-ms` may be set to account for estimated latency so that Prebid Server can handle timeouts from adapters and respond to the AMP RTC request before it times out.
```

Comment:
I don't quite understand a meaning of  _may be set to account_. In my implementation, it set on application config level.